.github/workflows/ci.yml

```diff
@@ -67,3 +67,57 @@ jobs:
 
       - name: Unit Test
         run: yarn test
+
+  generate-swagger:
+    runs-on: ubuntu-latest
+    env:
+      SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn config get cacheFolder)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Setup node and cache dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Nest Build
+        run: yarn build
+
+      - name: Install swaggerhub-cli
+        run: yarn global add swaggerhub-cli
+
+      - name: Read package version
+        uses: tyankatsu0105/read-package-version-actions@v1
+        id: package-version
+
+      - name: Generate Swagger JSON
+        run: yarn generate-swagger
+        env:
+          APP_SWAGGER_Version: ${{ steps.package-version.outputs.version }}
+
+      - name: Upload on Swaggerhub - Cophr_develop
```

Comment:
recommend to align naming with below (`test`)

Reply:
I revised it in this commit. [264c533](https://github.com/Cophr/backend/pull/9/commits/264c533aa3e1dd913b184803ec32a21da0b1e7c2)
You are right. Thank you review.